src/smc_sagews/smc_sagews/tests/runtests.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+export SAGE_ROOT=/ext/sage/sage
+. /ext/sage/sage/src/bin/sage-env
```

Comment:
well, I'm wondering what this does, doctests, ok. ... I'll read the code :sunglasses: 

however, maybe make this line `. $SAGE_ROOT/src/bin/sage-env`

Reply:
Yes, of course. Done.